Bio/SeqIO/SffIO.py

```diff
@@ -230,10 +230,10 @@
 """
 
 
-from Bio.SeqIO.Interfaces import SequenceWriter
 from Bio import Alphabet
 from Bio.Seq import Seq
 from Bio.SeqRecord import SeqRecord
+from Bio.SeqIO.Interfaces import SequenceWriter
```

Comment:
If you are sorting alphabetically, it should be one line higher.

See also #2715 when I removed the ``import sys`` and put the standard library imports above the Biopython ones.